results/cmd/api/main.go

```diff
@@ -63,21 +63,47 @@ func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest
 	statement, err := database.Prepare("INSERT INTO taskrun (taskrunlog, uid, name, namespace) VALUES (?, ?, ?, ?)")
 	if err != nil {
 		log.Printf("failed to insert a new taskrun: %v\n", err)
-		return nil, fmt.Errorf("failed to insert a new taskrun: %v", err)
+		return nil, fmt.Errorf("failed to insert a new taskrun: %w", err)
 	}
 
 	// serialize data and insert it into database.
 	taskrunFromClient := req.GetTaskRun()
 	blobData, err := proto.Marshal(taskrunFromClient)
 	if err != nil {
 		log.Println("taskrun marshaling error: ", err)
-		return nil, fmt.Errorf("failed to marshal taskrun: %v", err)
+		return nil, fmt.Errorf("failed to marshal taskrun: %w", err)
 	}
 	taskrunMeta := taskrunFromClient.GetMetadata()
 	if _, err := statement.Exec(blobData, taskrunMeta.GetUid(), taskrunMeta.GetName(), taskrunMeta.GetNamespace()); err != nil {
 		log.Printf("failed to execute insertion of a new taskrun: %v\n", err)
-		return nil, fmt.Errorf("failed to excute insertion a new taskrun: %v", err)
+		return nil, fmt.Errorf("failed to excute insertion a new taskrun: %w", err)
 
 	}
 	return taskrunFromClient, nil
 }
+
+// Get TaskRun from database and send it to watcher
+func (s *server) GetTaskRun(ctx context.Context, req *pb.GetTaskRunRequest) (*pb.TaskRun, error) {
+	database := s.db
```

Comment:
the database variable is only used once. why not change the next line to s.db.Query() and get rid of the database variable?

Reply:
done